src/CLI/renderer/templates/show_portchannel.j2

```diff
@@ -0,0 +1,9 @@
+{% set just_var = 2 %}
+
+Flags: A - Active, Dw - Oper status down
```

Comment:
Any reason for not following the below show format?

S6K-12# show interface port-channel summary
LAG     Mode      Status    Uptime              Ports
10      L2        up        1 day 15:05:13      Eth 1/1/1 (Up)
S6K-12# show interface port-channel
Port-channel 10 is up, line protocol is up
Address is 90:b1:1c:f4:9c:c0, Current address is 90:b1:1c:f4:9c:c0
Interface index is 52
Internet address is not set
Mode of IPv4 Address Assignment: not set
Interface IPv6 oper status: Disabled
MTU 1532 bytes, IP MTU 1500 bytes
LineSpeed 40G
Minimum number of links to bring Port-channel up is 1
Maximum active members that are allowed in the portchannel is 32
Members in this channel: Eth 1/1/1
ARP type: ARPA, ARP Timeout: 60
Last clearing of "show interface" counters: 1 weeks 5 days 02:40:03
Queuing strategy: fifo
Input statistics:
     2430414 packets, 180817217 octets
     2346588 64-byte pkts, 457 over 64-byte pkts, 19 over 127-byte pkts
     83350 over 255-byte pkts, 0 over 511-byte pkts, 0 over 1023-byte pkts
     2430036 Multicasts, 0 Broadcasts, 378 Unicasts
     0 runts, 0 giants, 0 throttles
     0 CRC, 0 overrun, 0 discarded
Output statistics:
     2570312 packets, 209118628 octets
     140055 64-byte pkts, 2346864 over 64-byte pkts, 39 over 127-byte pkts
     83354 over 255-byte pkts, 0 over 511-byte pkts, 0 over 1023-byte pkts
     2569933 Multicasts, 6 Broadcasts, 373 Unicasts
     0 throttles, 1 discarded, 0 Collisions,  wred drops
Rate Info(interval  seconds):
     Input 0 Mbits/sec, 0 packets/sec, 0% of line rate
     Output 0 Mbits/sec, 1 packets/sec, 0% of line rate
Time since last interface status change: 1 day 15:05:24


S6K-12# show interface port-channel 10
Port-channel 10 is up, line protocol is up
Address is 90:b1:1c:f4:9c:c0, Current address is 90:b1:1c:f4:9c:c0
Interface index is 52
Internet address is not set
Mode of IPv4 Address Assignment: not set
Interface IPv6 oper status: Disabled
MTU 1532 bytes, IP MTU 1500 bytes
LineSpeed 40G
Members in this channel: Eth 1/1/1
ARP type: ARPA, ARP Timeout: 60
Last clearing of "show interface" counters: 1 weeks 5 days 02:40:15
Queuing strategy: fifo
Input statistics:
     2430415 packets, 180817583 octets
     2346588 64-byte pkts, 457 over 64-byte pkts, 19 over 127-byte pkts
     83351 over 255-byte pkts, 0 over 511-byte pkts, 0 over 1023-byte pkts
     2430037 Multicasts, 0 Broadcasts, 378 Unicasts
     0 runts, 0 giants, 0 throttles
     0 CRC, 0 overrun, 0 discarded
Output statistics:
     2570323 packets, 209119642 octets
     140065 64-byte pkts, 2346864 over 64-byte pkts, 39 over 127-byte pkts
     83355 over 255-byte pkts, 0 over 511-byte pkts, 0 over 1023-byte pkts
     2569944 Multicasts, 6 Broadcasts, 373 Unicasts
     0 throttles, 1 discarded, 0 Collisions,  wred drops
Rate Info(interval  seconds):
     Input 0 Mbits/sec, 0 packets/sec, 0% of line rate
     Output 0 Mbits/sec, 1 packets/sec, 0% of line rate
Time since last interface status change: 1 day 15:05:36


S6K-12#

Reply:
After looking further into IS-CLI supported commands and based the info I am getting from Redis-DB I will be only supporting the command "show port-channel summary" .  Have added the change in HLD also.